stream-access-control/access-control-core/src/main/java/com/backbase/stream/service/UserService.java

```diff
@@ -237,15 +241,49 @@
      * @return {@link Mono<Void>}
      */
     public Mono<User> updateIdentityUserAttributes(User user) {
-        if (user.getAttributes() == null)
-            Mono.just(user);
+        if (IdentityUserLinkStrategy.IMPORT_FROM_IDENTIY.equals(user.getIdentityLinkStrategy())) {
+            if (user.getAttributes() == null)
+                Mono.just(user);
```

Comment:
isn't it missing a return here?

Reply:
`IdentityUserLinkStrategy.IMPORT_FROM_IDENTIY.equals(user.getIdentityLinkStrategy()) && user.getAttributes() != null`